### DISCUSSION TITLE
restore back up  of users to new server

### DISCUSSION THREAD

**Discussion by farshad-fallah** [BearWare/TeamTalk5#1880] [General]
how i can restore xml files include users  from old server to new server  in cent os 7

 with which command?
thanks

**Comment by bear101** [comment]
copy the old xml file to a new one, e.g. tt5srv.1.xml. Then run "`tt5srv -c tt5srv.1.xml -wizard`" to reconfigure.

**Reply by farshad-fallah** [reply]
thanks i did that but i get  invalid account  error  for old user and new user i have been created
and question 2:  i must creat new file or new directory for new tt5srv.xml file
 
thanks before

**Comment by farshad-fallah** [comment]
sorry i am hear because  i ask and reply  to your answer in git hob  but
no one  answerd  me

thanks i did that but i get invalid account error for old user and new user
i have been created
and question 2: i must creat new file or new directory for new tt5srv.xml
file

**Reply by bear101** [reply]
tt5srv executable will default to tt5srv.xml if you don't specify the -c option. Maybe that's your problem so you keep loading tt5srv.xml instead of tt5srv.1.xml.
